src/toil_scripts/lib/files.py

```diff
@@ -49,6 +53,49 @@ def copy_file_job(job, name, file_id, output_dir):
     copy_files([fpath], output_dir)
 
 
+def copy_to(filename, output_dir, work_dir=None):
+    """
+    Moves files from the working directory to the output directory.
```

Comment:
This description doesn't agree with the logic in the function that checks for absolute paths, etc.